src/websocket.rs

```diff
@@ -100,6 +201,9 @@ impl AdminWebsocket {
                 }
                 info!("returning newly created agent key");
                 self.agent_key = Some(key.clone());
+                // if using devNet,
+                // enable membrane proof using generated key
+                self.enable_memproof_dev_net();
```

Comment:
you need to pass the agent_key that is generated here to be signed and used in the mem-proof